src/plugins/irc-events/link.js

```diff
@@ -16,10 +16,9 @@ module.exports = function(irc, network) {
 		}
 
 		var links = [];
-		var split = data.message.split(" ");
+		var split = data.message.replace(/\x02|\x1D|\x1F|\x16|\x0F|\x03(?:[0-9]{1,2}(?:,[0-9]{1,2})?)?/g, "").split(" ");
 		_.each(split, function(w) {
-			var match = w.indexOf("http://") === 0 || w.indexOf("https://") === 0;
-			if (match) {
+			if (/^https?:\/\//.test(w)) {
```

Comment:
I'm not going to block this, but in general I'm not in favor of obfuscating a couple checks with a regex. It's shorter, yes, but regexes are arguably harder to understand and, when not backed with tests, tend to let bugs go through more easily.
The previous check was a tiny bit longer but more straightforward to read (granted, storing this in that `match` variable was unnecessary).